app/src/main/AndroidManifest.xml

```diff
@@ -31,10 +31,11 @@
         <activity android:name=".AddSaleActivity"
             android:parentActivityName=".MainActivity"/>
 
-        <activity android:name=".ListSalesActivity"
-            android:parentActivityName=".MainActivity"></activity>
-        <activity android:name=".LoginActivity"></activity>
-        <activity android:name=".UserProfileActivity"></activity>
+        <activity android:name=".LoginActivity"
+            android:parentActivityName=".MainActivity"/>
+
+        <activity android:name=".UserProfileActivity"
+            android:parentActivityName=".MainActivity"/>
 

```

Comment:
The arrow button on the sign up page exit the app instead of going back to mainActivity. I'm not sure how you're handling that but my hunch is that because SignUpActivity is not in your manifest (not even without a parent)

Reply:
This is fixed in my next PR for Registration by email/password. This PR don't really use the signUpActivity that's why it is neglected here.